reinforcement_learning/rlclientlib/factory_resolver.cc

```diff
@@ -89,26 +89,28 @@ namespace reinforcement_learning {
     return error_code::success;
   }
 
-  int observation_sender_create(i_sender** retval, const u::configuration& cfg, i_trace* trace_logger, api_status* status) {
+  int observation_sender_create(i_sender** retval, const u::configuration& cfg, error_callback_fn* _error_cb, i_trace* trace_logger, api_status* status) {
     *retval = new eventhub_client(
       cfg.get(name::OBSERVATION_EH_HOST, "localhost:8080"),
       cfg.get(name::OBSERVATION_EH_KEY_NAME, ""),
       cfg.get(name::OBSERVATION_EH_KEY, ""),
       cfg.get(name::OBSERVATION_EH_NAME, "observation"),
       cfg.get_int(name::OBSERVATION_EH_TASKS_LIMIT, 16),
       trace_logger,
+      _error_cb,
       cfg.get_bool(name::EH_TEST, false));
     return error_code::success;
   }
 
-  int interaction_sender_create(i_sender** retval, const u::configuration& cfg, i_trace* trace_logger, api_status* status) {
+  int interaction_sender_create(i_sender** retval, const u::configuration& cfg, error_callback_fn* _error_cb, i_trace* trace_logger, api_status* status) {
     *retval = new eventhub_client(
       cfg.get(name::INTERACTION_EH_HOST, "localhost:8080"),
       cfg.get(name::INTERACTION_EH_KEY_NAME, ""),
       cfg.get(name::INTERACTION_EH_KEY, ""),
       cfg.get(name::INTERACTION_EH_NAME, "interaction"),
       cfg.get_int(name::INTERACTION_EH_TASKS_LIMIT, 16),
       trace_logger,
+      _error_cb,
```

Comment:
[Minor][Nit] _error_cb => error_cb (+others)

Not important enough to block this PR. Please fix in followup.